tests/feature_resolution_editions.js

```diff
@@ -0,0 +1,106 @@
+/*
+1. Defaults
```

Comment:
While I agree that this monolithic test proto is ideal for initial development, can we split it up into fine-grained protos?  That would make the tests more readable and not require copy/pasting the entire thing into comments :P. Giving more meaningful feature names might help too (e.g. FILE, MESSAGE, FIELD, etc).

I'm not 100% clear on how protobufjs works, but I think you could also define the proto inline in the test instead of a separate file.  That would make each test much more self-contained and easier to read